scripts/opentitan/tests-passing.txt

```diff
@@ -84,6 +88,11 @@
 //sw/device/tests:example_concurrency_test_sim_qemu_rom_with_fake_keys
 //sw/device/tests:example_test_from_flash_sim_qemu_rom_with_fake_keys
 //sw/device/tests:flash_ctrl_clock_freqs_test_sim_qemu_rom_with_fake_keys
+//sw/device/tests:flash_ctrl_info_access_lc_dev_sim_qemu_rom_with_fake_keys
+//sw/device/tests:flash_ctrl_info_access_lc_prod_end_sim_qemu_rom_with_fake_keys
+//sw/device/tests:flash_ctrl_info_access_lc_prod_sim_qemu_rom_with_fake_keys
+//sw/device/tests:flash_ctrl_info_access_lc_rma_sim_qemu_rom_with_fake_keys
+//sw/device/tests:flash_ctrl_info_access_lc_test_unlocked0_sim_qemu_rom_with_fake_keys
```

Comment:
We briefly discussed this in person, but the OpenTitan side doesn't change the OTP for these so they might not be all passing